src/main/java/com/gruzini/tennistico/domain/Player.java

```diff
@@ -11,26 +11,25 @@
 import javax.validation.constraints.NotNull;
 import java.time.LocalDate;
 import java.time.Period;
-import java.util.List;
 
 @Entity(name = "players")
 @Data
 @NoArgsConstructor
 @AllArgsConstructor
 @Builder
 public class Player {
-   @Id
-   @GeneratedValue(strategy = GenerationType.IDENTITY)
-   private Long id;
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE)
+    private Long id;
 
-   @NotNull
-   private String firstName;
+    @NotNull
+    private String firstName;
 
-   @NotNull
-   private String lastName;
+    @NotNull
+    private String lastName;
```

Comment:
Tak tylko informacyjnie: u siebie na branchu już dodałem getter getFullName(), który zwraca imię i nazwisko jako jeden String. Myślę, że przyda się to do dodania do treści notyfikacji.